test/image/compare_pixels_test.js

```diff
@@ -97,7 +97,9 @@ function untestableFilter(mockName) {
         mockName === 'font-wishlist' ||
         mockName.indexOf('gl2d_') !== -1 ||
         mockName.indexOf('mapbox_') !== -1 ||
-        mockName.indexOf('gl3d_cone-') !== -1
+        mockName.indexOf('gl3d_cone-') !== -1 ||
+        mockName.indexOf('gl3d_volume_between-ranges') !== -1 ||
```

Comment:
Ok. We should add those `gl3d_volume` mocks to

https://github.com/plotly/plotly.js/blob/02e51d8f5ba5601f53ee3dc839ba05fb12c7f8a2/tasks/noci_test.sh#L31-L39

also.